docs/identity/saas-apps/veza-tutorial.md

```diff
@@ -158,29 +144,29 @@ In this section, you'll enable B.Simon to use single sign-on by granting access
 
     b. Open the downloaded **Certificate (Base64)** and upload the file into the **X509 Signing Certificate** by clicking **Choose File** option.
 
-    c. In the **Sign Out Url** textbox, paste the **Logout URL** value, which you've copied.
```

Comment:
I've removed the step to set the Sign Out URL, as this is no longer in the product.